<launch>
	<param name="image_raw" 	value="/camera/color/image_raw" />
	<param name="camera_info" 	value="/camera/color/camera_info" />
	
	<param name="object_name"	value="cube" />
		
	<node pkg="visp_robotis_tracker" type="visp_robotis_tracker" name="visp_robotis_tracker" output="screen">
	</node>
</launch>